src/devices/Media/SoundDevice/Devices/UnixSoundDevice.cs

```diff
@@ -5,9 +5,11 @@
 using System;
 using System.Buffers.Binary;
 using System.IO;
+using System.Runtime.CompilerServices;
```

Comment:
are you using the new namespaces?

Reply:
it's not needed, will remove. I did few tests and should be removed.